README.md

```diff
@@ -340,6 +340,7 @@ Examples exist on [codesandbox.io][examples]:
 
 - [downshift Apollo example](https://codesandbox.io/s/j2omZpK3W)
 - [downshift Spectre.css example](https://codesandbox.io/s/M89KQOBRB)
+- [downshift GenieJS example](https://codesandbox.io/s/4zNL2WW1)
```

Comment:
Sorry, my bad. I thought to create a similar example based on `genie` with downshift rather than importing and using the library.

@kentcdodds, can you please point me in the right direction? How can I import `geniejs`?

I have tried in `codesandbox`:
* `import 'geniejs'`
* `import Genie from 'geniejs'`
* `require('geniejs')`
* `import 'genie'`
* `import Genie from 'genie'`
* `require('genie')`

Then doing in my `App` `render` the following:
```
    const trashWish = geniejs({
      magicWords: 'Take out the trash',
      action: function() {
        console.log('Yes! I love taking out the trash!');
      }
    });
```
I keep getting `geniejs` is not defined. I have saved one of my tries in [here](https://codesandbox.io/s/4zNL2WW1) for your reference.